airbyte/cli.py

```diff
@@ -447,6 +452,93 @@ def benchmark(
     )
 
 
+@click.command()
+@click.option(
+    "--source",
+    type=str,
+    help=(
+        "The source name, with an optional version declaration. "
+        "If the name contains a colon (':'), it will be interpreted as a docker image and tag. "
+    ),
+)
+@click.option(
+    "--destination",
+    type=str,
+    help=(
+        "The destination name, with an optional version declaration. "
+        "If a path is provided, it will be interpreted as a path to the local executable. "
+    ),
+)
+@click.option(
+    "--streams",
+    type=str,
+    help=(
+        "A comma-separated list of stream names to select for reading. If set to '*', all streams "
+        "will be selected. Defaults to '*'."
+    ),
+)
+@click.option(
+    "--Sconfig",
+    "source_config",
+    type=str,
+    help="The source config. " + CONFIG_HELP,
+)
+@click.option(
+    "--Dconfig",
+    "destination_config",
+    type=str,
+    help="The destination config. " + CONFIG_HELP,
+)
+@click.option(
+    "--Spip-url",
+    "source_pip_url",
+    type=str,
+    help="Optional pip URL for the source (Python connectors only). " + PIP_URL_HELP,
+)
+@click.option(
+    "--Dpip-url",
+    "destination_pip_url",
+    type=str,
+    help="Optional pip URL for the destination (Python connectors only). " + PIP_URL_HELP,
+)
+def sync(
+    *,
+    source: str,
+    source_config: str | None = None,
+    source_pip_url: str | None = None,
+    destination: str,
+    destination_config: str | None = None,
+    destination_pip_url: str | None = None,
+    streams: str | None = None,
```

Comment:
This was intentional. This will later be resolved to '*' but a future version would change the default to all "suggested" streams instead of "all" streams. If I hard-code '*' as a default, it will be harder to give this QoL improvement later.